### ISSUE TITLE
Sidebar and document view are being selected (highlighted) during scroll (in Safari/GNOME Web)

### ISSUE THREAD

**Issue by PeterHagen** [CollaboraOnline/online#6241]
**Describe the bug**
On scrolling a document with the sidebar open, when the mouse moves over the sidebar, the html elements get selected (sidebar and document view)

**To Reproduce**
1. Open a scrollable document
2. Open the sidebar
3. Scroll down the document with the scrollbar
4. While scrolling move the mouse over the sidebar

**Expected behavior**
Nothing should happen accept scrolling

**Actual behavior**
The HTML elements in the sidebar and the document view are being selected (highlighted)

**Screenshots**
<img width="825" alt="Screenshot 2023-04-27 at 15 12 55" src="https://user-images.githubusercontent.com/12300899/234872609-dad12b80-99c4-4f25-8126-e021c0b1f932.png">

**Desktop (please complete the following information)**
 - Device: MacBook
 - OS:  MacOS 13.3.1
 - Browser: Safari
 - Version: 16.4

**Optional solution**
Add the following css to the body of the cool.html page:

```
-moz-user-select: -moz-none;
-khtml-user-select: none;
-webkit-user-select: none;
user-select: none;
```

**Comment by Ezinnem** [comment]
Scrolling seems to work just as expected on my end.


@PeterHagen, Please can you share the Collabora Online version where you experienced this issue on?

**Comment by Ezinnem** [comment]
I tested using Gnome Web browser and this seems fixed using:

COOLWSD version:
23.05.1.2snapshot git hash: [[ https://github.com/CollaboraOnline/online/commits/9187ae9 | 9187ae9 ]](E)

Fixing commit: https://github.com/CollaboraOnline/online/commit/f56694ef9630f2d697aee9b777eea14ed9a4b817

**Comment by PeterHagen** [comment]
On Safari this is not yet solved on this version: 23.05.1.3 git hash: 10deb70
